server/plugin/command.go

```diff
@@ -358,7 +369,13 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 	} else if ghRepo != nil && ghRepo.GetPrivate() {
 		msg += "\n\n**Warning:** You subscribed to a private repository. Anyone with access to this channel will be able to read the events getting posted here."
 	}
-
+	githubHooks, _, _ := githubClient.Repositories.ListHooks(ctx, owner, repo, &github.ListOptions{PerPage: page})
+	if len(githubHooks) == 0 {
+		msg += "\nNo webhook was found for this repository or organization. Would you like the webhook to be created?"
+		msg += "\nSuggested Command, Can run any one command to create a wehook"
+		msg += fmt.Sprintf("\n`/github webhook add %s [secret:replace with webhook secret from Plugin console] [content-type:application-json/application/x-www-form-urlencoded] [features:create/delete..]  [ssecure-ssl:true/false]`", owner)
+		msg += fmt.Sprintf("\n`/github webhook add %s/%s [secret:replace with webhook secret from Plugin console] [content-type:application-json/application/x-www-form-urlencoded] [features:create/delete..]  [ssecure-ssl:true/false]`", owner, repo)
```

Comment:
@hanzei Should we put the secret here in the output, or have the admin go to the system console as noted here?

Reply:
@mickmister @hanzei 
made a change where the secret is picked directly from github plugin console , no need to pass it explicitly

```
   /github webhook add sibasankarnayak/mattermost-server  callback_url
```

Note : callback url is only need to pass it runs on localhost as github doesn't permit to add localhost url

Reply:
If the site URL is configured to use `localhost`, can we just return an error saying "please set your site URL to something other than localhost". The secret and callback URL will then be optional parameters, then we can keep the command nice and short.

Reply:
@mickmister currently site url will give  error if it is local host and secret is not at all needed  and it is optional
picking up from plugin console directly

Reply:
@sibasankarnayak Can we make it so the `callback_url` is optional? I was suggesting we assume that the site URL is not localhost in the main case, so the admin would not need to supply a `callback_url`.

The only time they need to supply a `callback_url` is if the site URL _is_ localhost. So let's not make them supply the `callback_url` when the site URL is not localhost.

Reply:
@mickmister current the callback url is optional , you can try it if your ulr is not local host